apps/backend/src/modules/ai-assistant/libs/helpers/initial-chat/balance-analysis.validation-schema.ts

```diff
@@ -0,0 +1,19 @@
+import { z } from "zod";
+
+const category = z.object({
+	categoryId: z.number(),
+	categoryName: z.string(),
+});
+
+const Messages = z.object({
```

Comment:
```suggestion
const messages = z.object({
```
Please, check others as well, when fixing the issue in one place